src/ert/dark_storage/common.py

```diff
@@ -163,21 +155,52 @@ def data_for_key(
     return pd.DataFrame()
 
 
+@dataclasses.dataclass
+class _ObservationWithXAxis:
+    obs_name: str
+    values: List[float]
+    errors: List[float]
+    x_axis: List[str]
+
+
+def get_observation_for_response(
+    ensemble: Ensemble, response_key: str
+) -> Optional[_ObservationWithXAxis]:
+    if "@" in response_key:
```

Comment:
Does this still occur by the way?

Reply:
I think so yep, we use the `@` notation in the plotter, and it is split thereafter. It does receive a name with `@` when running `test_direct_dark_performance_with_storage`